src/uct/sm/cma/cma_md.c

```diff
@@ -6,36 +6,119 @@
 */
 
 #define _GNU_SOURCE
-#include <sys/uio.h>
 #include "cma_md.h"
 
+#include <ucs/sys/string.h>
+#include <sys/prctl.h>
+#include <sys/uio.h>
+#include <string.h>
+
+#if HAVE_SYS_CAPABILITY_H
+#  include <sys/capability.h>
+#endif
+
+
 uct_md_component_t uct_cma_md_component;
 
-static ucs_status_t uct_cma_query_md_resources(uct_md_resource_desc_t **resources_p,
-                                               unsigned *num_resources_p)
+static int uct_cma_test_ptrace_scope()
```

Comment:
`() != (void)`, because `uct_cma_test_ptrace_scope(foo, bar)` is not an error in case of the former